reproductions/StackExchange.Redis.StackOverflowException/StackExchange.Redis.StackOverflowException.csproj

```diff
@@ -7,7 +7,6 @@
 
   <ItemGroup>
     <PackageReference Include="StackExchange.Redis" Version="2.0.601" />
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.7.0" />
```

Comment:
This reference was causing a nuget warning due to mismatched package version. We didn't need the direct reference here, anyway.